### PR TITLE
Bump resolutions dependencies

### PR DESCRIPTION
**Pull request by torchiaf**
Signed-off-by: Francesco Torchia [francesco.torchia@suse.com](mailto:francesco.torchia@suse.com)

### Summary
Contributes to #4534

- ejs to ^3.1.7,
- json5 to >=2.2.2
- d3-color to >=3.1.0
- glob-parent to >=5.1.2
- node-forge to >=1.3.0
- qs to >=6.7.3
- nth-check to >=2.0.1
- follow-redirects to >=1.14.7
- merge to >=2.1.1

Fixes `critical` and `high` vulnerabilities; `axios` and `loader-utils` are not updated at the moment.

### CODE REVIEW

File: shell/utils/axios.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import Axios from 'axios';
+import Axios, { CancelToken, isCancel } from 'axios';
```
Comment (mantis-toboggan-md) [inline shell/utils/axios.js:1]: Are these changes still needed now that axios is not being bumped in this PR?

Comment (torchiaf) [reply shell/utils/axios.js]: Not needed and removed. Also updated the PR description.
